exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/filter/FilterRecordBatch.java

```diff
@@ -164,15 +167,14 @@ protected Filterer generateSV4Filterer() throws SchemaChangeException {
     } catch (ClassTransformationException | IOException e) {
       throw new SchemaChangeException("Failure while attempting to load generated class", e);
     }
-
   }
 
   protected Filterer generateSV2Filterer() throws SchemaChangeException {
     final ErrorCollector collector = new ErrorCollectorImpl();
     final List<TransferPair> transfers = Lists.newArrayList();
     final ClassGenerator<Filterer> cg = CodeGenerator.getRoot(Filterer.TEMPLATE_DEFINITION2, context.getOptions());
-    // Uncomment below lines to enable saving generated code file for debugging
-    // cg.getCodeGenerator().plainJavaCapable(true);
+    cg.getCodeGenerator().plainJavaCapable(true);
```

Comment:
Is this change needed? Maybe it was left by mistake...

Reply:
The rule is supposed to be that all operators that support "plain Java" declare that they can do so. This allows us, at some point, to flip a switch and use the Java compiler rather than Janino and byte code fixup. So, the "plainJavaCapable" is supposed to be uncommented. Only the "saveCodeForDebugging" should be commented. So, this was fixing an earlier error.